Products/Zuul/facades/triggersfacade.py

```diff
@@ -319,12 +319,15 @@ def getTriggerList(self):
         response, trigger_set = self.triggers_service.getTriggers()
         trigger_set = to_dict(trigger_set)
         triggerList = []
+        user = getSecurityManager().getUser()
         if 'triggers' in trigger_set:
             for t in trigger_set['triggers']:
```

Comment:
Also this `if` before `for` is redundant, you can do: `for t in trigger_set.get('triggers', [])`